lib/SILOptimizer/SILCombiner/SILCombinerCastVisitors.cpp

```diff
@@ -978,10 +978,8 @@ SILInstruction *SILCombiner::visitConvertEscapeToNoEscapeInst(
       SILType::getPrimitiveObjectType(NewTy));
 }
 
-SILInstruction *SILCombiner::visitConvertFunctionInst(ConvertFunctionInst *CFI) {
-  if (CFI->getFunction()->hasOwnership())
-    return nullptr;
-
+SILInstruction *
+SILCombiner::visitConvertFunctionInst(ConvertFunctionInst *cfi) {
```

Comment:
I'll need to come back to reviewing `SILCombiner::visitConvertFunctionInst` later